weekly-openmeteo/requirements.txt

```diff
@@ -1,5 +1,5 @@
 requests
-psycopg2
+psycopg2-binary
```

Comment:
Important change, running the container without the psycopg in binary format can lead to issues.